cmd/containerd/main.go

```diff
@@ -148,6 +149,17 @@ func main() {
 }
 
 func before(context *cli.Context) error {
+	if context.GlobalString("config") == "default" {
+		fh, err := ioutil.TempFile("", "containerd-config.toml.")
```

Comment:
i considered, but opted for no, because what if you updated `containerd`, and a field changed. Now you want to compare existing config with the output of the new default. Let the admin decide that.